controllers/challenge.js

```diff
@@ -40,6 +41,20 @@ const sendChallengeResponse = async (req, res) => {
   }
 }
 
+/**
+ * @param {Array} allChallenges
+ * @returns {Promise<participants|Array>}
+ */
+const getParticipantsofChallenges = async (allChallenges) => {
+  return await Promise.all(allChallenges.map(async (challenge) => {
```

Comment:
What about the case when allChallenges is [] (empty)?

Reply:
I checked when we pass [] empty array it returns a 200 status with the message.

Reply:
cool